src/vm/frames.h

```diff
@@ -2267,6 +2276,23 @@ class StubDispatchFrame : public FramedMethodFrame
 
     Interception GetInterception();
 
+    virtual BOOL SuppressParamTypeArg()
+    {
+        MethodDesc *pMD = GetFunction();
+
+        //
+        // Shared default interface methods (i.e. virtual interface methods with an implementation) require
+        // an instantiation argument. But if we're in the stub dispatch frame, we haven't actually resolved
+        // the method yet (we could end up in class's override of this method, for example).
+        //
+        // So we need to pretent that unresolved default interface methods are like any other interface
+        // methods and don't have an instantiation argument.
+        //
+        // See code:CEEInfo::getMethodSigInternal
+        //
+        return pMD->GetMethodTable()->IsInterface();
```

Comment:
Are there any cases where we actually get into `StubDispatchFrame` with hidden generic arg? 

Maybe this can just return `TRUE` unconditionally.

Reply:
Normal virtual method calls on generic types maybe? Or they don't go through here?

Reply:
Normal virtual method calls do not have hidden generic arg. The context is in `this` pointer.

Generic virtual method calls do not go through here.

Reply:
I will leave it up to you whether to do this simplification.